PullToRefreshAnimation/PullToRefreshAnimation/SampleViewController.m

```diff
@@ -150,9 +131,10 @@ - (void)startLoading
 {
     self.isLoading = YES;
     
-    // start the loading animation
-    [self.loadingShape addAnimation:[self loadingAnimation]
-                             forKey:@"Write that word"];
+    // start the remaining part of the loading animation
+    self.loadingShape.timeOffset = 0.0;
+    self.loadingShape.speed = 1.0;
+    self.loadingShape.beginTime = [self.loadingShape convertTime:CACurrentMediaTime() fromLayer:nil] - 1.0;
```

Comment:
At this point 1 second has passed by controlling the timeOffset, so we set the beginTime to 1 second in the past and continue the animation at regular speed.